src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -30,7 +31,7 @@ def _process_sp_name(sp_name):
 class AzureKubernetesServiceScenarioTest(ScenarioTest):
 
     @AllowLargeResponse()
-    @ResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
```

Comment:
Why not change the value of `location='westus2'` in original code to use a location that works for the test? Using a environment variable hides the information about the valid location value.

Reply:
I'm going to deploy a runner to test aks commands with azure-cli in all locations where services are available and monitor the behavior of that. So I need to change the resource group location programmatically and I do not find other ways to change the default value of this decorator.

Reply:
Any suggestions to programmatically change the default value of location in the decorator are extremely welcome! Currently, I only come up with the method of passing an environment variable. If the ```AZURE_CLI_TEST_DEV_RESOURCE_GROUP_LOCATION``` variable is not manually set, it will not have any impact on existing tests.